spec/pragma.dd

```diff
@@ -272,7 +272,11 @@ $(H3 $(LNAME2 mangle, $(D pragma mangle)))
 
     $(P Overrides the default mangling for a symbol.)
 
-    $(P There must be one $(ASSIGNEXPRESSION) and it must evaluate at compile time to a string literal.
+    $(P For variables and functions there must be one $(ASSIGNEXPRESSION) and it must evaluate at compile time to a string literal.
+        For aggregates there may one or two $(ASSIGNEXPRESSION)s, one of which must evaluate at compile time to a string literal,
+        one which must evaluate to a symbol. If that symbol is a TemplateInstance the aggregate is treated as being a template
+        with the signature and arguments of the TemplateInstance. If no string is supplied the identifier of the symbol is used
```

Comment:
"If no string is supplied the identifier of the symbol is used" -> "The identifier of the symbol is used when no string is supplied."